anms-ui/public/app/components/management/agents/reports.vue

```diff
@@ -66,19 +67,24 @@ export default {
             console.error("reports error", error);
             console.info("error obj:", error);
           });
-      } else{
-        this.tableHeaders = this.reportsHeader[this.selected];
-        this.tableItems =  this.reports[this.selected];
-      }
+      // } else{
+        // this.tableHeaders = this.reportsHeader[this.selected];
+        // this.tableItems =  this.reports[this.selected];
+      // }
       
       this.loading = false;
     },
     processReport(report) {
-      this.tableHeaders = report.shift();
+      let holdHeader = report.shift();
+      this.tableHeaders = [];
+      for (let i = 0; i < holdHeader.length; i++) {
+        // this.tableHeaders.push({"key":holdHeader[i], "sortable":true});
```

Comment:
Same as above

Reply:
this was supposed to be commented/ not included since having the table sortbale breaks the stickyheaders. ill just remove it